Makefile

```diff
@@ -44,6 +44,9 @@ check:
 build: setup
 	bazel $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) //...
 
+fmt:
+	@bin/fmt.sh
```

Comment:
side note I think it's easier to debug without @ in front of commands so one knows what exact command is ran (I just changed a bunch of them, it's more important when there are variables/arguments so this one is maybe ok as is)